server/setup.py

```diff
@@ -48,6 +48,7 @@
         'jina>=3.12.0',
         'prometheus-client',
         'open_clip_torch>=2.8.0',
+        'cn_clip',
```

Comment:
I prefer to install it in the extra requirement

Reply:
already moved